Mathlib/Data/Finsupp/Interval.lean

```diff
@@ -114,7 +114,8 @@ end Lattice
 
 section CanonicallyOrdered
 
-variable [AddCommMonoid α] [PartialOrder α] [CanonicallyOrderedAdd α] [LocallyFiniteOrder α]
+variable [AddCommMonoid α] [PartialOrder α] [CanonicallyOrderedAdd α]
+  [OrderBot α] [LocallyFiniteOrder α]
```

Comment:
Does `LocallyFiniteOrderBot` work here? They're the same mathematically, but this isn't consistent with what you did in `Mathlib/Algebra/Order/Antidiag/Prod.lean`